client/knowledge_base_search/src/containers/SearchContainer/styles.scss

```diff
@@ -138,7 +138,9 @@
     background-color: color('teal', 40);
     color: color('white', 10);
     background-image: url(triangle(color('white', 10), $size: 1.5));
-    padding-right: 1.7rem;
+    width: 11rem;
```

Comment:
my gut feeling is that you should not have to modify these at all (and different screen sizes will probably have issues) but I have not pulled this down to check locally yet